README.md

```diff
@@ -1,8 +1,71 @@
-# home-financials — build instructions
+# Home Financials
 
-This project uses CMake. The top-level `CMakeLists.txt` requests C++23 for modern compilers.
+A terminal-based personal financial management application for tracking family finances, managing bank accounts, and computing net worth.
 
-Quick build (out-of-source):
+## Overview
+
+Home Financials is a C++ application that helps you manage your household finances through a terminal user interface (TUI). It allows you to:
+
+- Organize finances by family groups and individual members
+- Import bank account statements (currently supports Canara Bank, with extensible support for other banks)
+- Track bank accounts and balances
+- Compute net worth for individual members and entire families
+- Store all data securely in a local SQLite database
+
+## Features
+
+- **Family Management**: Create and manage family groups with multiple members
+- **Member Management**: Add, update, and delete family members
+- **Bank Statement Import**: Import transaction data from bank statement CSV files
+- **Net Worth Calculation**: Automatically compute net worth based on imported bank account data
+- **Data Persistence**: All data stored in a local SQLite database (`homefinancials.db`)
+- **Terminal UI**: Clean, interactive terminal-based interface
+
+## Prerequisites
+
+To build and run Home Financials, you need:
+
+- **C++ Compiler**: Modern C++ compiler with C++23 support (GCC 11+, Clang 14+, or equivalent)
```

Comment:
Do not suggest the use of clang at all. This project has never been built with anything other than gcc.